tests/std/tests/P0355R7_calendars_and_time_zones_formatting/test.cpp

```diff
@@ -293,12 +293,42 @@ bool test_day_formatter() {
     assert(format(STR("{:*5%}"), day{200}) == STR("**200"));
     assert(format(STR("{}"), day{0}) == STR("00 is not a valid day"));
 
+    // Op <<
     stream_helper(STR("00 is not a valid day"), day{0});
     stream_helper(STR("27"), day{27});
     stream_helper(STR("200 is not a valid day"), day{200});
 
     return true;
 }
+
+
+template <typename CharT>
+bool test_month_formatter() {
+    assert(format(STR("{}"), month{1}) == STR("Jan"));
+    assert(format(STR("{}"), month{12}) == STR("Dec"));
+    assert(format(STR("{}"), month{0}) == STR("0 is not a valid month"));
+    assert(format(STR("{}"), month{20}) == STR("20 is not a valid month"));
+
+    // Specs
+    assert(format(STR("{%b %h %B}"), month{1}) == STR("Jan Jan January"));
+    assert(format(STR("{%m Om}"), month{1}) == STR("1 1"));

```

Comment:
Some of these expected values will need to be corrected when the tests actually run; e.g. `%m` says "The month as a decimal number. Jan is 01. If the result is a single digit, it is prefixed with 0. The modified command %Om produces the locale’s alternative representation."

Also, this is missing a `%` before `Om`.